doc/engsys_resource_management.md

```diff
@@ -0,0 +1,107 @@
+## Resource Management Guidelines
+
+This document contains guidelines for creating and managing Azure resources in the Azure SDK Engineering System
+subscriptions. EngSys has automation that will delete resources based on the criteria outlined below. This is to
+improve our overall security and keep us in compliance with company policies.
+
+  * [Managing Dev Resources in the Playground Subscription](#managing-dev-resources-in-the-playground-subscription)
+     * [Resource Groups](#resource-groups)
+     * [Role Assignments](#role-assignments)
+  * [Managing CI resources in the Test Subscription](#managing-ci-resources-in-the-test-subscription)
+     * [Resource Groups](#resource-groups-1)
+     * [Role Assignments](#role-assignments-1)
+
+## Managing Dev Resources in the Playground Subscription
+
+This section applies to resource groups located in the `Azure SDK Developer Playground` subscription.
+
+Currently, the automation will inspect resource groups and role assignments only.
+
+### Resource Groups
+
+The automation will delete all resource groups in the playground subscription DAILY, unless they meet at least one of the following criteria:
+
+- The resource group starts with a valid Microsoft alias within `microsoft.com` or `ntdev.microsoft.com`, followed by an
+  optional hyphen and extra characters.
+    - Valid group name examples: `myalias`, `myalias-test-foobar` for `myalias@microsoft.com`
```

Comment:
Did we decide to not support "rg-myalias"?

Reply:
I thought we had decided to support it in the cleanup scripts but not document it for manual resource creation? If someone is creating groups through the provisioning scripts, they will be prefixed with `rg-` but will also have `DeleteAfter` tags.

Reply:
OK we can leave out "rg-" from the docs but still handle it in the script. Actually we probably don't need to handle it at all if the goal is that they will have DeleteAfter tags too.

Reply:
Yeah it ends up being redundant because the scripts that follow the convention do both things, otherwise I'd be worried that people would infer their resource would be deleted when it actually was valid.

I guess there's one loophole here where if you name your group `rg-mygroup` and don't tag it, you could expect that it would be cleaned up when it wouldn't. I'm not worried about this edge case and doubt it will ever be hit.